auto-discovery/cloud-aws/templates/service/service.yaml

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: apps/v1
```

Comment:
This is adapted from the kubernetes AutoDiscovery, where the directory and file are namend _manager_ but since we figured that this is not really a manager/controller/operator in the kubernetes sense I changed all references to manager to _service_. I guess the directory should also get a different name then? What would a proper name be? Also deployment for the directory would work I guess?